docs/user/filenodes.md

```diff
@@ -0,0 +1,56 @@
+# FlowForge File Nodes
+
+Projects running within FlowForge include a set of nodes that make it possible
+to store files safely regardless of the environment. For edge devices the files 
+are stored locally, containerized applications can read and write to persistent 
+storage via API calls to a file-server instance by using these nodes.
+
+*NOTE: This set of nodes allow you to read and write persistent files in a containerised
+environment only when coupled with a [file-storage](../install/file-storage/README.md) instance*
+
+
+## Nodes
+
+There are two nodes in the File Node collection:
+
+- `file` - A file node for writing to persistent storage
+- `file in` - A file node for reading from persistent storage
+
+## Install
+
+### FlowForge
+
+The File Nodes are installed automatically with FlowForge however in order for a 
+project to use them, the built-in Node-RED file nodes must first be disabled. 
+This is done by simply adding `10-file-js` in the **Exclude nodes by filename** 
+section of your FlowForge projects settings under  the **Palette** section
+
+### Standalone Node-RED
```

Comment:
Why would a user go this route? I don't get this section at all.

Reply:
As the FlowForge instance may not be on FF Cloud, the FF admin / owner needs to know how to do this.

But you do raise an interesting point. There may be cases where they want to disable the node-red file nodes but NOT enable the cloud based file nodes.

Need to discuss with @hardillb - but this may be a next iteration thing.

Reply:
Current situation:

- File storage can be enabled/disabled globally for a FlowForge Instance by adding/removing the `fileStore` block in the `flowforge.yml`
- If enabled then disabling the core file nodes will enable the FlowForge File Nodes
- There is currently no way to have projects with a mix of some with node file nodes at all and some with the FlowForge Nodes.

I suggest we raise an item for the next release to implment disabling all File node support in the template if it's enabled at the FlowForge Insance leve.

Reply:
Story raised: https://github.com/flowforge/flowforge/issues/1472